root/treeformula/scan/runLoadDim.C

```diff
@@ -0,0 +1,7 @@
+#include "TChain.h"
+
+void runLoadDim() {
+   TChain c("Nominal/llllTree");
+   c.Add("my.output.root");
```

Comment:
```suggestion
   c.Add("loadDim.root");
```
Make the file name more clearer specific to the test.